config/locales/en.yml

```diff
@@ -75,9 +76,10 @@ en:
     applicant_name: Applicant
     reference: Ref. no.
     time_passed: Time passed
-    received_at: Date received
-    reviewed_at: Date reviewed
+    received_on: Date received
```

Comment:
Whats the reasoning behind changing these key names?

Reply:
It just made more sense in English for me. "Application received on 21st December" is more correct than "Application received at 21st December" - I think that at is for a time (or DateTime which is why it is like this in the db records) - I'm pretty indifferent if it's better to put them back as it was